### DISCUSSION TITLE
Validating new antibodies (optimal concentration of antibody, control tissues, etc)

### DISCUSSION THREAD

**Discussion by Moisesoagj** [IBEXImagingCommunity/ibex_imaging_knowledge_base#156] [Q&A] [answered]
Hello everyone,

Excuse me, I have some questions about how can I correctly validate new antibodies for their use on IBEX. I am working with human paediatric thymi and I want to analyse stromal and epithelial compartments.  Most of my antibodies are already validated. However, I have 4 new markers that have never been tested and are very important to characterize subpopulations of our interest. Therefore, I would like to ask you about the following points:

1. Titrate the antibody. Some experts have recommended starting with a base dilution of 1:50, and others 1:100. I have even considered it appropriate to use the one recommended by the manufacturer. But given this, based on your experience, which option do you recommend to me to take? and how many dilutions to test would be the ideal?
2. Controls. It is important to have a tissue that works as a positive control (positive for the marker), but would you consider appropriate other kind of controls?
3. When titrating, should I use one slide to test one marker at a time, or can I test several markers on that slide? 
4. Quality Control of the sample. Some experts recommend using a little panel with the most typical markers of the tissue. In my case I was considering just to include CD45, EpCAM, CD3 and Laminin A. What do you think about this?
5. Order of antibodies. Moreover, some people have recommended me to test the order of the antibodies. Could someone explain to me how to do that?

I thank you in advance, any kind of help will be very useful.

**Comment by radtkea** [comment, marked as answer]
Hi Moisesoagj!

Thank you for your EXCELLENT questions! So excited for your study in the thymus! It's an exquisite organ.

We recently published an [organ mapping antibody panel](https://www.nature.com/articles/s41592-023-01846-7) for the human thymus, [OMAP-17](https://humanatlas.io/omap). Hope this is helpful to you!

**1A. Titrate the antibody or "What concentration of antibody should I use for validation tests?"**
-Manufacturer recommendations _can_ be helpful, if listed. However, the manufacturer is unlikely to report your exact tissue preservation method, antigen retrieval conditions (if FFPE), and imaging set up. Instead, the concentration of the antibody, if available, can provide key details about the antibody formulation. 
-Example 1: This [Keratin 10](https://www.novusbio.com/products/cytokeratin-10-antibody-de-k10_nbp2-47650) antibody from Novus Biologicals lists a concentration of 0.2 mg/ml under _Packaging, Storage & Formulations_.
-Example 2: This [Keratin 5](https://www.biolegend.com/en-us/products/keratin-5-polyclonal-antibody-purified-10956) antibody from BioLegend lists a _Concentration_ of 1 mg/ml.
-The antibody in Example 2 is 5X more concentrated than Example 1, so a lower dilution (1:200) versus (1:40-50) could be used for pilot tests.
**-What I do:** When initially testing antibodies, I always start with A LOT of antibody (1:50 dilution, 10-15 ug/ml) because I would rather see something and then titrate accordingly. There is always a danger of a false negative from adding too little antibody. 

**1B. Titrate the antibody or "How many titrations do you recommend?"**
Due to the tedious nature of imaging, I don't exhaustively titrate antibodies like with flow cytometry. The biggest things to consider when titrating your antibodies is 1) optimal signal to noise and 2) minimal spectral overlap. I find that I have to do more careful titrations with adjacent fluorophores and dim and bright markers. In OMAP-17 Cycle 5, Aquaporin 1 AF647 was SUPER bright and spilling into Ki-67 AF700. We had to titrate Aquaporin 1 at least 5 times to arrive at 1:3000 to be compatible with the 1:25 dilution for Ki-67. Antibody dilutions can be highly dependent on the panel and not just the individual antibody, if that makes sense? Happy to discuss/advise via email (andrea.radtke "at" nih.gov). 

**2. Controls** 
Great question! Positive controls are always needed for antibody validation but how to determine the best controls? As an example, when we worked up our OMAP-17 thymus panel we could use other lymphoid tissues (lymph node and spleen) to validate most of the immune and structural markers. The thymus itself could be used to validate markers that are found in abundance in the thymus and have a well established labeling pattern in the tissue, defined positive and negative cell types, and subcellular location. For example, FOXP3 would be enriched in the medulla and found in the nuclei of CD3+CD4+ T cells. Markers used to define rare mTEC populations required other tissues for validation. We used the pancreas to validate chromogranin A. Vendor sheets can help identify good positive control tissue. See the example images for this [Chromogranin A antibody](https://www.novusbio.com/products/chromogranin-a-antibody-lk2h10_nbp2-34672). [Human Protein Atlas](https://www.proteinatlas.org/ENSG00000100604-CHGA/tissue) is tremendously helpful for understanding the spatial distribution of a marker. I understand that it is not always possible to get other tissues prepared according to your specifications. This is why community antibody validation efforts are so helpful such as the IBEX Imaging Community and OMAPs. Hopefully, you can find an antibody against your marker of interest validated in either your tissue or a different tissue prepared using the same fixation and antigen retrieval conditions. When in doubt about the best control tissues for your marker, ASK the discussion forum! We will help:)

**3. One slide per marker or multiple markers per slide**
This depends on the antibodies and the microscope you will be using. In general, I prefer to use simple panels (3-4 fluorophores per tissue section max) in well separated fluorophores: Hoechst, AF488, AF555, and AF647 when initially testing antibodies. While our more sophisticated microscopes can separate AF532/AF555, AF555/AF594, and AF647/AF700, spectral overlap often occurs between these channels when antibodies are improperly titrated in the initial pilot tests (See 1A). Another consideration is whether these antibodies are conjugated or require secondary antibodies to visualize. 
-Example 1: With these three antibodies [Chromogranin A](https://www.novusbio.com/products/chromogranin-a-antibody-lk2h10_nbp2-34672), [Keratin 5](https://www.biolegend.com/en-us/products/keratin-5-polyclonal-antibody-purified-10956), and [Lumican] (https://www.rndsystems.com/products/human-lumican-antibody_af2846), you could design a panel with a donkey anti-mouse IgG AF488 to detect Chromogranin A, a donkey anti-rabbit IgG AF555 to detect Keratin 5, and a donkey anti-goat IgG AF647 to detect Lumican. 
-Example 2: If testing more than one unconjugated mouse, rabbit, goat, etc primary antibody, then you would have to separate across tissue sections. Yes, it is possible to do different anti-mouse isotype antibodies. I have used that approach in OMAP-17 (anti-IgM and anti-IgG, the order matters) and other projects. Happy to share my favorite anti-isotype secondary antibodies with you.

**4. Quality control panel**
Yes, this is critical! I always recommend screening tissues with a small panel that covers major anatomical structures and cell types in your tissue of interest. This gives you a "lay of the land" and allows for critical evaluation of the tissue before IBEX imaging. This saves you from wasting time and resources on bad tissue. It will be dependent on the tissue you are imaging. 

**5. Order of antibodies**
Yes, antibody order can impact antibody performance. For this reason, I evaluate the performance of the panel on individual serial sections in parallel with IBEX imaging. Discussed in [Figure S5](https://www.pnas.org/doi/full/10.1073/pnas.2018488117).
-Simple 2 Cycle IBEX experiment example: From the same tissue block, cut multiple tissue sections. Apply Cycle 1 antibodies in the IBEX chamber. Cycle 1 does not need to test for steric hindrance or sensitivity to LiBH4 treatment. Put sensitive antibodies and unconjugated primary antibodies in this cycle. For Cycle 2 panel, apply half of the Cycle 2 antibody cocktail to the IBEX imaging chamber and the other half of the antibody cocktail to a serial tissue section that has not been labeled with Cycle 1 antibodies or dye inactivated with LiBH4. Compare the labeling pattern of antibodies visualized in serial versus IBEX imaged tissue. If comparable, then antibody can be in Cycle 2. If lost/diminished, then move to Cycle 1. 

I appreciate this is a lot of information and, unfortunately, a lot of it really depends on your tissue, your panel, your instrument, etc. Happy to walk through details specific to your experiments. Best of luck!

**Comment by Moisesoagj** [comment]
Dear Andrea,

This is an exceptional and very useful information. Exactly the answers to
what I had asked, many thanks. Regarding the quality control panel, I
designed the next one considering each anatomical structure of the thymus,
please let me know what do you think.

My best wishes :)

Moi

Cycle Chanel Marker Anatomical Structure Clone Dye Vendor Host/Isotype
Catalogue
1 488 Lamin A Nucleous 133A2 FITC Thermo Fisher Scientific Mouse / IgG3
 MUB1101L1
555 CD31/PECAM-1  Vessels  WM59 PE Thermofisher Mouse IgG1, κ 12-0319-42
594 CD3 Cortex and medulla stroma UCHT1 iF594 Caprico Biotechnologies Mouse
IgG1, κ 1053136
750 Lumican Capsule and trabeculas Polyclonal Customize/CoraLite® Plus 750 R&D
Systems  Goat/IgG AF2846
2 594 CD45 Cortex and medulla stroma F10-89-4 iF594 CapricoBio  Mouse IgG1,
κ 1016136
647 Keratin 8 Cortical Epithelium H1+TS1 AF647  Novus Biologicals Mouse
IgG1, κ NBP2-34627AF647
750 Keratin 5 Medullary Epithelium Poly19055 BL750 custom conjugate
BioLegend  Rabbit / IgG 905501

**Reply by radtkea** [reply]
Hi Moi:)

So happy it was helpful! Thank you for asking your questions openly in the discussion forum. 

Indeed, a panel that allows one to survey the major anatomical structures of a tissue is key. For our OMAPs, we ask all author to highlight 4-6 key markers to examine in their tissue. This serves as a quality control panel, in a way. [See Figure 2 part 1](https://www.nature.com/articles/s41592-023-01846-7/figures/2). 

This looks like a good panel!

- Lamin A: This will label the nuclear lamina of cells. I know from my friend/colleague Nadav, that the TEC populations of the thymus uniquely express this marker based on scRNA-seq. In our hands, this antibody varied from sample to sample. We think this is due to differences in tissue handling times but we don't really know. In summary, this marker can serve as indicator for thymus tissue quality, at least for us. However, it is not a critically important marker for assessing anatomical structures and cell types in the thymus. In our OMAP ([OMAP-17](https://humanatlas.io/omap)), we detail the following markers for the "core panel" in Column W with rationale provided in Column X: CD8, CD3, CD4, HLA-DR, DEC205, and Lumican. 

All the other markers you selected are excellent. You will have all immune cells (CD45), vessels (CD31), trabeculae/matrix (lumican), and key mTEC and cTEC markers (Keratin 5 and Keratin 8). 

Two suggestions:
1. Strongly recommend inclusion of DEC205 for the cortical TECs. We found that this cell type varied widely between samples. We excluded samples that had a notable absence of cTECs. I need to find some example images for you. I imagine Keratin 8 would be able to reveal these differences but I am not sure.
2. Your panel currently has: 
- Lamin FITC (AF488 channel)
- CD31 PE (AF555 channel)
- CD45 iF594 (AF594 channel)
- Keratin 8 AF647 (AF647 channel)
- Lumican AF750 + Keratin 5 BL750 (AF750 channel). These two antibodies will conflict.

An alternative:
- Lamin FITC (AF488 channel)
- [DEC205 ](https://www.novusbio.com/products/dec-205-cd205-antibody-jb87-35_nbp2-75467) with a donkey anti-rabbit AF555,  CoraLite 555 FlexAble kit (seems like you are using these!), or Zenon AF555 kit.
- CD45 iF594 (AF594 channel)
- HLA-DR AF647 (AF647 channel); This will give you all antigen presenting cells (DCs, B cells) and TECs. Beautiful! The L243 clone works gorgeous on fixed frozen samples and is available through many vendors. 
- Lumican AF750

Excited for your work! Please share your dye inactivation results with the community so we add share [here](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/fluorescent_probes.html). We don't have CoraLite 750 yet:)

Thank you and best wishes,
A

**Comment by Moisesoagj** [comment]
Dear Andrea :)

Considering the alternative options that you mentioned, and also
considering the need to test the antibodies after that strange situation
with the dry ice, I constructed the next panel for running the
Quality Controls and for testing the antibodies:

CD205 (FITC) [Cortex Epithelial Compartment]
CD31 (PE) [Vessels]
CD45 (AF594) General Stromal [General Thymic Stroma]
HLA-DR ( AF647) General Stromal [General Thymic Stroma]
Keratin 5 ( AF750) Medulla [Medullary Epithelial Compartment]

What do you think?

Best wishes and good start to the week,

Moi

**Comment by radtkea** [comment]
Hi Moi!

Thank you so much. I hope you are having a great start to your week!

Looks good! Minor edits (in italics):
CD205 (FITC) [Cortex Epithelial Compartment] _How will you visualize it with the FITC conjugate? Will you use the CoraLite 488 FlexAble kits or an anti-rabbit FITC secondary antibody?_ 
CD31 (PE) [Vessels]
CD45 (AF594) _Pan-immune marker not general stroma_
HLA-DR ( AF647) General Stromal [General Thymic Stroma]
Keratin 5 ( AF750) Medulla [Medullary Epithelial Compartment]

Best of luck and we will resolve the issue with the antibodies soon. Hopefully, they are all okay. 

Best wishes,
A

**Comment by Moisesoagj** [comment]
Hi Andrea,

Many thanks :) everything is going well, I also hope you are having a nice
week. Excellent recomendations. Let me speack with Alex to purchase the
CoraLite 488 kit to label CD205 as soon as possible and start working.
We've spoken with Colin and we are planning to run the first High Quality
Control sample in 10-15 days ( beacause I need to complete a fire induction
to have access to the UCL Institute of Oftalmology). So, the revamped panel
for these Quality Control samples is:

CD205 (CoraLite 488) [Cortex Epithelial Compartment]
CD31 (PE) [Vessels]
CD45 (AF594) [*Pan-immune marker]*
HLA-DR ( AF647) [General Thymic Stroma]
Keratin 5 ( AF750) [Medullary Epithelial Compartment]

And many thanks for that Andrea, we will be updating you on next weeks. I
also hope that all Abs are perfect 🙌🏼.

Best wishes,

Moi
